src/pages/NewChatPage.tsx

```diff
@@ -266,9 +294,9 @@ function NewChatPage({betas, isGroupChat, personalDetails, reports, isSearchingF
                             shouldShowConfirmButton
                             shouldShowReferralCTA={!dismissedReferralBanners?.[CONST.REFERRAL_PROGRAM.CONTENT_TYPES.START_CHAT]}
                             referralContentType={CONST.REFERRAL_PROGRAM.CONTENT_TYPES.START_CHAT}
-                            confirmButtonText={selectedOptions.length > 1 ? translate('newChatPage.createGroup') : translate('newChatPage.createChat')}
+                            confirmButtonText={selectedOptions.length > 1 ? translate('common.next') : translate('newChatPage.createChat')}
                             textInputAlert={isOffline ? [`${translate('common.youAppearToBeOffline')} ${translate('search.resultsAreLimited')}`, {isTranslated: true}] : ''}
-                            onConfirmSelection={createGroup}
+                            onConfirmSelection={selectedOptions.length > 1 ? navigateToConfirmPage : createChat}
```

Comment:
We should have removed the requirement that a group chat needs more than 1 other participant to be created because this requirement was only applicable in the `NewChatPage` and could be skipped in `NewChatConfirmPage ` causing inconsistency. Also because this was not really a requirement after all.